prometheus_client/multiprocess.py

```diff
@@ -56,7 +56,7 @@ def merge(files, accumulate=True):
                     metric.add_sample(name, labels_key, value)
             d.close()
 
-        for metric in metrics.values():
+        for n, metric in metrics.iteritems():
```

Comment:
`n` defined, but never used. I think it's worth returning `metrics.values ()`

Reply:
Good catch.  Fixed